etna/core/serializers/images.py

```diff
@@ -0,0 +1,121 @@
+from rest_framework.serializers import Serializer
+
+
+class ImageSerializer(Serializer):
+    """
+    This ImageSerializer was created to improve the `ImageRenditionField` that
+    comes as part of the Wagtail API. This serializer allows us to create a
+    JPEG and WEBP rendition in one field, rather than having to have multiple fields.
+
+    rendition_size defaults to `fill-600x400`, but can be specified
+    when the serializer is used, e.g:
+    ImageSerializer(rendition_size="original")
+
+    jpeg_quality and webp_quality default to 60 and 80 respectively,
+    and can be specified in the same way as rendition_size.
+
+    The source of the image can also be set in the serializer, e.g:
+    APIField("image_large", serializer=ImageSerializer(rendition_size="fill-900x900", source="image"))
+
+    This will come back as `image_large` in the API response.
+    """
+
+    def __init__(
+        self,
+        rendition_size="fill-600x400",
+        jpeg_quality=60,
+        webp_quality=80,
+        *args,
+        **kwargs,
+    ):
+        self.jpeg_quality = jpeg_quality
+        self.webp_quality = webp_quality
+        self.rendition_size = rendition_size
+        super().__init__(*args, **kwargs)
+
+    def to_representation(self, value):
+        if value:
+            jpeg_image = value.get_rendition(
+                f"{self.rendition_size}|format-jpeg|jpegquality-{self.jpeg_quality}"
+            )
+            webp_image = value.get_rendition(
+                f"{self.rendition_size}|format-webp|webpquality-{self.webp_quality}"
+            )
+
+            return {
+                "id": value.id,
+                "title": value.title,
+                "image_jpeg": {
```

Comment:
Maybe this just needs to be `jpeg` because this block always appears in an `image` property?

Reply:
Same with the `image_webp` -> `webp`?